### ISSUE TITLE
Force push in our own repo

### ISSUE THREAD

**Issue by 2011eric** [NTU-SP/SP-Hw1-release#48]
Will that be a problem?

**Comment by imingasco** [comment]
No, if you do so before deadline. If you do so after deadline, TAs won't be able to checkout to correct commit, and thus you may be regarded as late submission after next deadline checkpoint.
